ui/src/shared/middleware/resizeLayout.js

```diff
@@ -12,5 +14,12 @@ export default function resizeLayout() {
       evt.initEvent('resize', false, true)
       window.dispatchEvent(evt)
     }
+
+    const qs = queryString.parse(window.location.search)
+
+    if (typeof qs.present !== 'undefined') {
```

Comment:
much better, but one more minor thing: `http://localhost:8888/sources/2/dashboards/45?present=false` still yields presentation mode. can we fix this for the case that a user thinks of this as an explicit boolean, and thus it would toggle presentation mode with either `present` or `present=true` but not for `present=false`?

Reply:
Good point. This begs the question, then; should we allow for two interfaces? A presence check and a boolean check? Or just the boolean check? Because, I do agree, the boolean check makes sense, but the presence check might be a contrived and improper pattern.

Reply:
Upon further reflection, just the boolean check is a perfectly fine interface for this option, as well as it's documented somewhere, in the very least, in the changelog.

Reply:
I think the boolean interface is more expectable, though I like the presence once as well. I'd say, at least boolean, and presence if you want it. The current state of the code, as a boolean, I think is perfectly acceptable and sufficient.